contracts/foundry.toml

```diff
@@ -2,5 +2,7 @@
 src = 'src'
 out = 'out'
 libs = ['lib']
+solc_version = '0.8.18'
+test = 'tests'
```

Comment:
Are these two fields necessary for slither to work?

Reply:
the first one was recommended to be set, but should be fine without. The second one is needed so slither doesn't run on test files and bombard us with alerts in the PRs.